knownprojects_build/sql/dob_match.sql

```diff
@@ -0,0 +1,45 @@
+drop table if exists dob_review;
+with combined_dob as (
+    SELECT 
+    b.source, 
+    b.project_id::text, 
+    b.project_name, 
+    b.project_status, 
+    b.project_type,
+    b.number_of_units::integer, 
+    b.date, 
+    b.date_type, 
+    b.dcp_projectcompleted, 
+    null as portion_built_by_2025, 
+    null as portion_built_by_2035, 
+    null as portion_built_by_2055, 
+    a.cluster_id, 
+    a.sub_cluster_id, 
+    null as adjusted_units,
+    b.inactive,
+    b.geom
+    FROM combined a
+    JOIN dcp_housing b
+    ON st_intersects(a.geom, b.geom)
+    AND split_part(split_part(a.date, '/', 1), '-', 1)::numeric - 1 < extract(year from b.date::timestamp)
+    union
+    select * from combined),
+multimatch as (
+    select project_id
+    from combined_dob
+    where source = 'DOB'
+    group by project_id
+    having count(*) > 1),
+multimatchcluster as (
+	select distinct cluster_id
+	from combined_dob 
+	where project_id in (
+		select project_id 
+		from multimatch))
+select *, 
+    (case when cluster_id in 
+        (select cluster_id from multimatchcluster) then 1 
+        else 0 end) as review_flag 
+        into dob_review
+	from combined_dob
+	order by cluster_id, sub_cluster_id;
```

Comment:
I wonder if the table would be easier to review with `order by review_flag, cluster_id, sub_cluster_id`. I can't think of an easy way to put the multiple options for a single DOB job next to eachother, since we are trying to fit both types of reviews into one table (both verifying the one-to-ones and resolving the one-to-manys).